aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -47,9 +49,9 @@ struct Vec256 {
     Vec256 vec;
     for (int64_t i = 0; i < size; i++) {
       if (i < count) {
-        vec.values[i] = b.values[i];
+        vec.set_value(i, b[i]);
       } else {
-        vec.values[i] = a.values[i];
+        vec.set_value(i, a[i]);
```

Comment:
How is that better than accessing `values`?

Reply:
See comment aboe